docs/basic_usage.rst

```diff
@@ -3,6 +3,7 @@ Basic Usage
 
```

Comment:
You can just copy the changes to the README here.

P.S. If you have any idea of a better way to keep these two files in sync, let me know.

Reply:
Done and will do.